schema/sqitch.plan

```diff
@@ -5,6 +5,7 @@
 schema_ggircs_portal 2019-07-11T18:30:51Z Hamza Javed <hamza@button.is> # Add a schema for the GGIRCS Industry Portal (GIP)
 function_update_timestamps [schema_ggircs_portal] 2019-10-01T21:42:55Z Alec Wenzowski <alec@button.is> # A function to be used as a trigger to update the created_at and updated column values
 type_ciip_user_organisation_status [schema_ggircs_portal] 2019-11-06T21:07:19Z Helen Liu <helen@button.is> # added new type file ciip_user_organisation_status
+type_ciip_application_status [schema_ggircs_portal] 2019-11-26T18:39:57Z Helen Liu <helen@button.is> # Added ciip application status type
```

Comment:
`type_ciip_application_status` is used in the `table_application_status`, so you should add it as a dependency there (you can just edit this file directly and add the new dependency

Reply:
do you mean just declare the type in the application status table?

Reply:
I mean add `type_ciip_application_status` between the square brackets on line 20